tests/drivers/spi/spi_slave_master_loopback/prj_nucleo_f091rc.conf

```diff
@@ -0,0 +1,13 @@
+CONFIG_BOOT_BANNER=y
```

Comment:
What do you think about following the pattern in the led_ws2812 and led_lpd8806 samples (in samples/drivers) to have a top-level prj.conf, and merge in a board-specific fragment?

That will make it easier to add additional boards and have a clean top-level prj.conf.

Reply:
Good idea, I will do that